.node-version

```diff
@@ -0,0 +1 @@
+10.11.0
```

Comment:
Any particular reason why we're locking the node version to this one? I don't think that this file / locking the version is necessary given that this library works with any node 8+

Reply:
yeah no, just an artifact of me switching between different projects. i'll rm it